sdk/dataprotection/Microsoft.Azure.Management.DataProtection/Microsoft.Azure.Management.DataProtection.sln

```diff
@@ -0,0 +1,51 @@
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Management.DataProtection.Backup", "src\Microsoft.Azure.Management.DataProtection.Backup.csproj", "{D8F50755-3477-4D0F-B959-482F329997C1}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Azure.Management.DataProtection.Backup.Tests", "tests\Microsoft.Azure.Management.DataProtection.Backup.Tests.csproj", "{004937D0-0E95-47FA-A9D9-949D6224B79B}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Debug|x64 = Debug|x64
+		Debug|x86 = Debug|x86
+		Release|Any CPU = Release|Any CPU
+		Release|x64 = Release|x64
+		Release|x86 = Release|x86
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{D8F50755-3477-4D0F-B959-482F329997C1}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{D8F50755-3477-4D0F-B959-482F329997C1}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{D8F50755-3477-4D0F-B959-482F329997C1}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{D8F50755-3477-4D0F-B959-482F329997C1}.Debug|x64.Build.0 = Debug|Any CPU
+		{D8F50755-3477-4D0F-B959-482F329997C1}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{D8F50755-3477-4D0F-B959-482F329997C1}.Debug|x86.Build.0 = Debug|Any CPU
+		{D8F50755-3477-4D0F-B959-482F329997C1}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{D8F50755-3477-4D0F-B959-482F329997C1}.Release|Any CPU.Build.0 = Release|Any CPU
+		{D8F50755-3477-4D0F-B959-482F329997C1}.Release|x64.ActiveCfg = Release|Any CPU
+		{D8F50755-3477-4D0F-B959-482F329997C1}.Release|x64.Build.0 = Release|Any CPU
+		{D8F50755-3477-4D0F-B959-482F329997C1}.Release|x86.ActiveCfg = Release|Any CPU
+		{D8F50755-3477-4D0F-B959-482F329997C1}.Release|x86.Build.0 = Release|Any CPU
+		{004937D0-0E95-47FA-A9D9-949D6224B79B}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{004937D0-0E95-47FA-A9D9-949D6224B79B}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{004937D0-0E95-47FA-A9D9-949D6224B79B}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{004937D0-0E95-47FA-A9D9-949D6224B79B}.Debug|x64.Build.0 = Debug|Any CPU
+		{004937D0-0E95-47FA-A9D9-949D6224B79B}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{004937D0-0E95-47FA-A9D9-949D6224B79B}.Debug|x86.Build.0 = Debug|Any CPU
+		{004937D0-0E95-47FA-A9D9-949D6224B79B}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{004937D0-0E95-47FA-A9D9-949D6224B79B}.Release|Any CPU.Build.0 = Release|Any CPU
+		{004937D0-0E95-47FA-A9D9-949D6224B79B}.Release|x64.ActiveCfg = Release|Any CPU
+		{004937D0-0E95-47FA-A9D9-949D6224B79B}.Release|x64.Build.0 = Release|Any CPU
+		{004937D0-0E95-47FA-A9D9-949D6224B79B}.Release|x86.ActiveCfg = Release|Any CPU
+		{004937D0-0E95-47FA-A9D9-949D6224B79B}.Release|x86.Build.0 = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		SolutionGuid = {7BCCAB5F-5599-4596-A0CC-4E49ECD39DEF}
+	EndGlobalSection
+EndGlobal
```

Comment:
remove 'upgraxdelog.htm' file below

Reply:
removed and added package in pipeline yaml